web/packages/teleterm/src/services/config/getKeyboardShortcutSchema.ts

```diff
@@ -104,11 +113,17 @@ function validateKeyCodeAndModifiers(
       return z.NEVER;
     }
 
-    expectedModifiers.forEach(expectedModifier => {
-      if (!allowedModifiers.includes(expectedModifier)) {
-        ctx.addIssue(invalidModifierIssue(expectedModifier));
-      }
-    });
+    const invalidModifiers = expectedModifiers.filter(
+      expectedModifier => !allowedModifiers.includes(expectedModifier)
+    );
```

Comment:
Why do we use the `expected` prefix here? I was trying to figure out what happens here and I thought `expectedModifiers` is some version of supported modifiers or something.

I think we could drop the prefix, couldn't we?